app/admin/forms.py

```diff
@@ -8,7 +8,7 @@
 
 def select_field_choices():
     choices = [('Admin', 'Admin'),
-               ('regularUser', 'Regular User'),
+               ('Regular User', 'Regular User'),
```

Comment:
Is the first field here, "Regular User", that used to be "regularUser", the id name of the selected option by the user or something like that?  If it's the id name, shouldn't this be a single word and not a value with a space in it?

Reply:
yeah, the first argument is the value and the 2nd is what's displayed in the dropdown menu. On the model I entered the check constraint as "Regular User". I added the space because that value in "security_permissions" of the User model is being displayed in the admin portal and the user profile, so that it can show up as "Regular User". I didn't want to write some Javascript to add a space between the words and capitalize the 'r' to switch from "regularUser" to "Regular User" when it renders in the HTML, and I thought that was a good enough reason lol.